velox/exec/tests/PlanNodeToSummaryStringTest.cpp

```diff
@@ -96,6 +96,13 @@ TEST_F(PlanNodeToSummaryStringTest, basic) {
           .maxOutputFields = 3,
           .maxChildTypes = 2,
       }));
+  ASSERT_EQ(
```

Comment:
Please, add empty line before ASSERT_EQ.

Please, add similar checks to other tests.